examples/getting_started/quickstart.ipynb

```diff
@@ -0,0 +1,535 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "os.environ[\"OPENAI_API_KEY\"] = \"YOUR_API_KEY\"\n",
+    "# Set the appropriate API key for the provider you're using"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prompt Templates\n",
+    "\n",
+    "Prompt templates in Mirascope allow you to create dynamic and reusable prompts."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-09-06T14:00:06.593677Z",
+     "start_time": "2024-09-06T14:00:06.590420Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "None\n"
+     ]
+    }
+   ],
+   "source": [
+    "from mirascope.core import prompt_template\n",
+    "\n",
+    "\n",
+    "@prompt_template(\"What is the capital of {country}?\")\n",
+    "def get_capital_prompt(country: str): ...\n",
+    "\n",
+    "\n",
+    "print(get_capital_prompt(\"Japan\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example:\n",
+    "1. The `@prompt_template` decorator defines the structure of the prompt.\n",
+    "2. `{country}` is a placeholder that gets replaced with the matching argument passed to the function.\n",
+    "\n",
+    "Here's an example of a multi-line prompt:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-09-06T14:00:08.173878Z",
+     "start_time": "2024-09-06T14:00:08.170209Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "None\n"
+     ]
+    }
+   ],
+   "source": [
+    "@prompt_template(\n",
+    "    \"\"\"\n",
+    "    SYSTEM: You are a helpful assistant that provides information about {topic}.\n",
+    "    USER: What are the key points to understand about {subtopic} in {topic}?\n",
+    "    \"\"\"\n",
+    ")\n",
+    "def get_info_prompt(topic: str, subtopic: str): ...\n",
+    "\n",
+    "\n",
+    "print(get_info_prompt(\"artificial intelligence\", \"machine learning\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This example:\n",
+    "1. Includes \"SYSTEM\" and \"USER\" keywords to define messages with different roles in the prompt.\n",
+    "2. Uses multiple variables (`{topic}` and `{subtopic}`) to create a more specific and dynamic prompt.\n",
+    "\n",
+    "Prompt templates improve code readability, maintain consistency in prompts, and make it easier to experiment with and modify prompts for effective prompt engineering.\n",
+    "\n",
+    "For more detailed information on prompt templates, check out our [Learn documentation on Prompts](https://docs.mirascope.io/learn/prompts)."
```

Comment:
nit: I think we drop "Learn" and just have the link be "documentation on Prompts"

I would do the same for all of the other links as well